apps/src/code-studio/pd/application_dashboard/summary_table.jsx

```diff
@@ -108,7 +105,6 @@ export class SummaryTable extends React.Component {
         {
           this.props.data.accepted.locked + this.props.data.accepted.unlocked > 0 && (
             <Button
-              href={this.context.router.createHref(`/${this.props.path}_cohort`)}
```

Comment:
Because we do the same thing in `handleViewCohortClick`. We only need one or the other, so whichever we prefer the style of can stay

Reply:
They accomplish different goals. I'm pretty sure we want both.

 1. href make it appear as a link in the browser, so you get a url preview in the bottom and can do things like right click to open in a new tab.
 2. the on click event handler marshals an click to go through React-Router in-client rather than actually navigating to the href url. 

True we don't need both, but it provides a better experience.

Reply:
oh i see, that makes sense!